src/composables/use-media-query.js

```diff
@@ -13,6 +13,7 @@ export function useMediaQuery(query, options = {}) {
   if (!window) return ref(false)
 
   const mediaQuery = window.matchMedia(query)
+  /** @type {import('@nuxtjs/composition-api').Ref<Boolean>} */
```

Comment:
Minor type nit pick: `Boolean` (with a capital B) refers to the primitive constructor, `boolean` with a lowercase b refers to the general type. Same goes for the rest of the primitive types.
```suggestion
  /** @type {import('@nuxtjs/composition-api').Ref<boolean>} */
```

Reply:
D'oh! I always do this with `Boolean` in particular, thanks!